transport-classes-io_uring/src/main/java/io/netty/channel/uring/CompletionQueue.java

```diff
@@ -42,19 +42,21 @@ final class CompletionQueue {
     final long ringAddress;
     final int ringFd;
     final int ringEntries;
+    final int ringCapacity;
```

Comment:
This field should be used in `IoUringIoHandler` to create the `CompletionBuffer`